src/cli.ts

```diff
@@ -4,119 +4,43 @@
  * Module dependencies.
  */
 import * as program from "commander";
-import chalk from "chalk";
-import { setupUnibeautify, findInstalledBeautifiers, getSupportedLanguages } from "./index";
+
 const pkg = require("../package.json");
+import { Runner, IArgs } from "./Runner";
 
 program
   .version(pkg.version)
-  .command("beautify", "Beautify files", {isDefault: true})
+  .command("beautify", "Beautify files", { isDefault: true })
   .usage("[options] [files...]")
   .option("-l, --language <language>", "Language of file to beautify")
   // .option("-i, --input-file [file]", "Input file to be beautified")
-  .option("-f, --file-path <file path>", "Path of the file to beautify from stdin")
+  .option(
+    "-f, --file-path <file path>",
+    "Path of the file to beautify from stdin"
+  )
   .option("-o, --out-file <file>", "Output file of beautified results")
   .option("-r, --replace", "Replace file(s) with beautified results", false)
   .option("-c, --config-file <file>", "Beautifier configuration file")
-  .option("--config-json <JSON>", "Beautifier configuration as stringified JSON")
-  // .option("-b, --beautifiers [beautifier...]", "Installed Beautifiers to load")
-  // .option("--input-dir [directory]", "Input directory of files to be beautified")
-  // .option("--output-dir [directory]", "Output directory of beautified results")
-  ;
+  .option(
+    "--config-json <JSON>",
+    "Beautifier configuration as stringified JSON"
+  );
+// .option("-b, --beautifiers [beautifier...]", "Installed Beautifiers to load")
+// .option("--input-dir [directory]", "Input directory of files to be beautified")
+// .option("--output-dir [directory]", "Output directory of beautified results")
 
 program
-  .command("list [options]", "List languages available or beautifiers installed")
+  .command("support", "List supported languages and/or beautifiers installed")
+  .alias("list")
   .alias("ls")
   .option("-l, --languages", "List available languages")
```

Comment:
Unfortunately commander doesn't separate options between the different commands you define (at least with my testing yesterday).  So we'll have to use something other than `-l` for this.  Vice versa we can't use `-b` in the beautify command in the future.

Reply:
This sounds wrong. I have yet to test though.

Reply:
Well, you can use them, but if you try `unibeautify list -l` it will error saying:

`error: option `-l, --language <language>' argument missing`